dbms/src/IO/WriteBufferFromS3.cpp

```diff
@@ -1,22 +1,18 @@
 #include <IO/WriteBufferFromS3.h>
 
-#include <IO/S3Common.h>
 #include <IO/WriteHelpers.h>
 
-#include <Poco/DOM/AutoPtr.h>
-#include <Poco/DOM/DOMParser.h>
-#include <Poco/DOM/Document.h>
-#include <Poco/DOM/NodeList.h>
-#include <Poco/SAX/InputSource.h>
-
 #include <common/logger_useful.h>
+#include <aws/s3/model/CreateMultipartUploadRequest.h>
```

Comment:
also `#ifdef`. Maybe we can create small wrapper around AWS API and place all this headers in one place (just proposal, not required)?